files.js/script.js

```diff
@@ -32,3 +32,12 @@ function hideMenu () {
 
 }
 
+// const parent = document.querySelector('.modal-parent'),
+//       btn = document.querySelector('button')
+//       X = document.querySelector('.close-icon');
+      
+// btn.addEventListener('click', show);
+
+// function show() {
+//   parent.style.display = 'block';
+// }
```

Comment:
- [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer